touchforms/formplayer/templates/touchforms/touchscreen.html

```diff
@@ -16,7 +16,7 @@
     <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/deprecate/formnav.js?version={{app_version}}"></script>
     <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/vendor/dates.js?version={{app_version}}"></script>
     <script language="javascript" type="text/javascript" src="{{STATIC_URL}}formplayer/script/deprecate/entrycontrols.js?version={{app_version}}"></script>
-    <script language="javascript" type="text/javascript;version=1.7" src="{{STATIC_URL}}formplayer/script/decprecate/workflow.js?version={{app_version}}"></script>
+    <script language="javascript" type="text/javascript;version=1.7" src="{{STATIC_URL}}formplayer/script/deprecate/workflow.js?version={{app_version}}"></script>
```

Comment:
No I don't think so. I really wanted to just delete it but wanted to chat to you first. If you access `/webforms/` locally you can get to this page.